src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/EmailTemplates.java

```diff
@@ -63,6 +63,9 @@
     @NotEmpty
     private String offerCounterSignedEmailToOtherParty;
 
+    @NotEmpty
+    private String freeMediationAcceptedEmailToClaimant;
```

Comment:
Would it be possible to have one template with optional content? Notify documentation says:

```
Optional content

Use double brackets and ‘??’ to define optional content.

For example if you only want to show something to people who are under 18:

((under18??Please get your application signed by a parent or guardian.))
For each person you send this message to, specify ‘yes’ or ‘no’ to show or hide this content.
```

Reply:
If not possible lets just rename property to be `claimantResponseWithMediationIssued` or to follow other pattern `responseWithMediationIssuedEmailToClaimant`. My point here is that the pain event is still response submission, `freeMediationAccepted` is a bit confusion for me.